packages/hyperproxy-node/src/index.js

```diff
@@ -35,33 +40,45 @@ export default class HyperproxyNode {
     /*
         Connect to the hub under the specified key.
     */
-    async _connectToHub(key) {
-        const client = new HyperproxyHubClient(WebSocket, Wrtc);
-        client.hub.subscribe(key).on('data', (data) => {
+    _connectToHub(key) {
+        this.client = new HyperproxyHubClient(WebSocket, Wrtc);
+        this.client.hub.subscribe(key).on('data', (data) => {
             try {
-                this._handleData(client, key, data);
+                this._handleData(key, data);
             } catch (e) {
                 console.error("ERROR in hyperproxy-node! ", err);
             }
         });
     }
 
+    _getPeerFrom(peerKey, peers) {
+        // peers.filter(peer => peer. == peerkKey)
```

Comment:
Remove commented out code? it seems like this function doesn't do anything. Should it still be here?